stream_script.py

```diff
@@ -146,16 +150,38 @@ def main():
     response = resp.json()
     print(response)
     tracker_id = response['tracker_instance_id']
+    
+    result_pipe = None
+    
     try:
+        if LIVEKIT_TOKEN_FOR_RESULTS is not None:
```

Comment:
is `LIVEKIT_TOKEN_FOR_RESULTS` None or "" by default?

Reply:
it's None if LIVEKIT_TOKEN_FOR_RESULTS is not in the .env file, but '' if it's in the file but just blank or just whitespace .. I guess I should have it default to '' and catch that then?

Reply:
maybe both - I could imagine user failure in either direction

Reply:
it's None because I'm saying in the .get to return None in the case where it's missing .. if I return '' in the case where it's missing I can catch both modes in one go I think

Reply:
yeah that sounds good!